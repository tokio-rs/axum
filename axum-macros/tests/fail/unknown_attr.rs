use axum_macros::FromRequest;

#[derive(FromRequest)]
struct Extractor(#[from_request(foo)] String);

fn main() {}
