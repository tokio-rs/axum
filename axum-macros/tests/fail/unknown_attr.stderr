error: expected `via`
 --> tests/fail/unknown_attr.rs:4:33
  |
4 | struct Extractor(#[from_request(foo)] String);
  |                                 ^^^
